GoogleForCreators/web-stories-wp
# Redesign: Media edit mode resize

## Context



## Summary



## Relevant Technical Choices



## To-do

- [x] Verify if it's OK to separate slider component into a separate issue (creating the component + switching to the new component in the edit mode).



## User-facing changes
The Media edit mode resize has been redesigned in this PR. Please see [Figma.](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=3616%3A64347)

Note: this PR does NOT include slider update since that component has not been created yet.


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a media element
2. Enter edit mode
3. Verify that the resize box looks like in Figma (see link above).

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6202
